src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+// eslint-disable-next-line no-unused-vars
+import { BrowserRouter as Router, Link } from 'react-router-dom'
```

Comment:
You only need to import BrowserRouter once per application, usually wrapping your whole app, like you did in your `App.js`. If you delete this, I guess you also don't need eslint in line 3.

Reply:
When I delete the import statement in `Navigation.js`, it fails to compile. I use the eslint-comment as Router is not used in the file. However, when I exclude it, the Links in the Navigation fail to respond to user input. Any ideas?
<img width="645" alt="image" src="https://user-images.githubusercontent.com/67916925/94121956-3d73e400-fe52-11ea-9805-e2f73e767502.png">

Reply:
Just importing like this does not work?
```js
import { Link } from 'react-router-dom'
```

Reply:
That works, thanks